### PR TITLE
fix(slider): allow slider to use text input when min is something oth…

### PR DESCRIPTION
**Pull request by tw15egan**
This fixes https://github.com/carbon-design-system/carbon-components-react/issues/287

**To test**: Change the `min` to something other than `0` and confirm you can change the slider value by changing the text input

### CODE REVIEW

File: src/components/Slider/Slider.js [hunk 1]
```diff
@@ -66,6 +66,11 @@ export default class Slider extends PureComponent {
 
     requestAnimationFrame(() => {
       this.setState((prevState, props) => {
+        if (evt && evt.target.classList.contains('bx-slider-text-input')) {
```
Comment (joshblack) [inline src/components/Slider/Slider.js]: Is it possible that `evt.target` could be undefined here?

Comment (asudoh) [reply src/components/Slider/Slider.js]: Another thing I'm wondering is if `evt.target` could be a SVG: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/1173756/

Comment (tw15egan) [reply src/components/Slider/Slider.js]: Would it be good to just change this to :

```js
if (evt && evt.target && evt.target.classList.contains('bx-slider-text-input'))
```

Comment (joshblack) [reply src/components/Slider/Slider.js]: Sounds good to me 😄

Comment (tw15egan) [reply src/components/Slider/Slider.js]: 👍

**Pull request merged**